scripts/sanity_chk/harness.py

```diff
@@ -78,14 +86,14 @@ def handle(self, line):
         elif self.GCOV_END in line:
             self.capture_coverage = False
 
+result_re = re.compile("(PASS|FAIL|SKIP) - (test_)?(.*)")
```

Comment:
Unrelated, but could do `(:?test_)` to turn that into a non-capturing group, since the "test_" text never needs to be extracted. `match.group(3)` below would then change into `match.group(2)`.

Doubt it makes any measurable performance difference though.